src/components/PokemonSprite.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PokeSprite from 'react-poke-sprites';
+import 'react-tippy/dist/tippy.css';
+import { Tooltip } from 'react-tippy';
+
+const Info = ({ info }) => {
+  return (
+    <React.Fragment>
+      <div>name: {info.species} </div>
```

Comment:
What other information would be interesting to look at? I will reformat the base stats display as it is just ugly json at the moment.

Reply:
What I was thinking of is 
`name, height, weight, eveything in base stats, and abilities`

Would that be too long? What do you think?